packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -181,6 +205,7 @@ abstract contract MajorityVotingBase is IMajorityVoting, MetaTxComponent, TimeHe
     /// @notice Internal function to execute a vote. It assumes the queried vote exists.
     /// @param _voteId The ID of the vote.
     function _execute(uint256 _voteId) internal virtual {
+        IDAO dao = getDAO();
```

Comment:
Why is this favorable? Doesn't this shadow the underlying variable `IDAO dao` that we have in `DaoAuthorizableBase`?
I assume this also creates a compiler warning because of this shadowing.